lib/main.dart

```diff
@@ -1,83 +1,149 @@
+// Import Dart standard libraries and Flutter packages
+import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:flutter_driver/driver_extension.dart';
 
+// Define a constant indicating whether the app is in drive mode
+// The value is obtained from the environment variables during compilation
+// ignore: constant_identifier_names
 const bool DRIVE_MODE = bool.fromEnvironment('DRIVE_MODE');
 
-Future<void> main() async {
-  // TASK: Read this value from a local storage
-  final useFlutterDriver = true;
-
-  if (!useFlutterDriver) {
-    WidgetsFlutterBinding.ensureInitialized();
-  } else {
-    enableFlutterDriverExtension();
-  }
-
+// Main function for the Flutter application
+void main() {
+  // Run the Flutter application
```

Comment:
I need to read the value here so I can decide if I should call 

```dart
WidgetsFlutterBinding.ensureInitialized();

// or 

enableFlutterDriverExtension();
```

I can only call one of those, and it has to be done before the app runs.

Reply:
Please check my updated PR. I think, I already full filled your requirement.